File: feinblelib/src/main/java/com/bbg/feinblelib/callback/BleRssiCallback.java
package com.bbg.feinblelib.callback;

import com.bbg.feinblelib.exception.BleException;

public abstract class BleRssiCallback extends BleBaseCallback{

    public abstract void onRssiFailure(BleException exception);

    public abstract void onRssiSuccess(int rssi);

}

File: feinblelib/src/main/java/com/bbg/feinblelib/callback/BleRssiCallback.kt
package com.bbg.feinblelib.callback

import com.bbg.feinblelib.exception.BleException

abstract class BleRssiCallback : BleBaseCallback() {
    abstract fun onRssiFailure(exception: BleException?)
    abstract fun onRssiSuccess(rssi: Int)
}